packages/atomic/src/components/common/generated-answer/styles/generated-answer.pcss

```diff
@@ -74,14 +74,18 @@
 [part='generated-container'] {
   &.answer-collapsed {
     @apply relative max-h-64 overflow-hidden content-[''];
+    max-height: var(--atomic-crga-collapsed-height, 16rem);
```

Comment:
In order for this new component-specific CSS variable to appear in the generated reference documentation, you'd need to add a comment with the @prop annotation.

E.g., https://github.com/coveo/ui-kit/blob/2b7404f4b2cfefaf1cef8d7e0d195ba282d4d7b6/packages/atomic/src/components/search/facets/atomic-color-facet/atomic-color-facet.pcss#L6-L9

is what generates the corresponding docs here:

https://docs.coveo.com/en/atomic/latest/reference/components/atomic-color-facet/#css-custom-content

Reply:
Another doc-related comment:

Should we update the documentation of the `collapsible` prop on the `atomic-generated-answer` component?

At the moment, it says:

>  Whether to allow the answer to be collapsed when the text is taller than 250px.

Perhaps it should say something like:

> Whether to allow the answer to be collapsed when the text is taller than the specified `--atomic-crga-collapsed-height` value (16rem by default).

CC @jpmarceau